### PR TITLE
Updates to Hook guides standardizing nutrition facts

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Review of all Hook guides to standardize on [nutrition facts](https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/2399470775/Nutrition+facts+in+developer+docs)
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-454773](https://oktainc.atlassian.net/browse/OKTA-454773)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/main/index.md [hunk 1]
```diff
@@ -10,17 +10,13 @@ This guide explains common set-up steps when implementing an Okta Event or Inlin
 
 **Learning outcomes**
 
-* Understand common set up steps for implementing an Okta Event or Inline Hook
-* Understand how to use these steps when running the example hook code in the accompanying guides
+* Understand common set up steps for implementing an Okta Event or Inline Hook.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/main/index.md:13]: I think "set-up" is a compound adjective, in this case.
```suggestion
* Understand common set-up steps for implementing an Okta Event or Inline Hook.
```

File: packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md [hunk 2]
```diff
@@ -26,6 +26,10 @@ This guide provides a working example of an Okta Event Hook. It uses the Hookdec
 
 ---
 
+## Overview
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md:29]: Most pages with nut facts use the "About..." structure. I think we should align and use either "about" or "overview"

File: packages/@okta/vuepress-site/docs/guides/password-import-inline-hook/main/index.md [hunk 3]
```diff
@@ -10,14 +10,14 @@ This guide provides a working example of an Okta Password Import Hook. It uses t
...
-* Preview and test a Password Import Inline Hook
+* Understand the Okta Password Import Inline Hook calls and responses.
+* Implement a simple working example of a Password Import Inline Hook with a Glitch.com project.
+* Preview and test a Password Import Inline Hook.
 
 **What you need**
 
-* A [Glitch.com](https://glitch.com) project or account.
-* An Okta developer org. [Create an org for free](https://developer.okta.com/signup/).
+* [Glitch.com](https://glitch.com) project or account
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/password-import-inline-hook/main/index.md:19]: Probably a very minor observation, but in some "Learning outcomes" sections for hooks, you start with the Okta org. In others, you start with "glitch" or "ngrok". Maybe for the sake of consistency, we could always start with the Okta org?

**Review by brentschaus-okta — APPROVED** [review]
LGTM

just a few minor observations and questions.

File: packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md (hunk 2 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md:29]: Thanks. Good catch. I'll use "About" and make the updates to all.

File: packages/@okta/vuepress-site/docs/guides/password-import-inline-hook/main/index.md (hunk 3 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/password-import-inline-hook/main/index.md:19]: Good eye! I'll start with the dev org first. Thanks.

**Review by brentschaus-okta — APPROVED** [review]
LGTMBBQ

**Pull request merged**